### ISSUE TITLE
Logcollector wildcard detects folders as logs

### ISSUE THREAD

**Issue by okynos** [wazuh/wazuh#3769]
Hello team,

|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 3.9.4-3932 | Logcollector | Agent | Packages | CentOS 7 |

We have detected a bad behavior on logcollector wildcards we will explain all details that we have:

**ossec.conf** (Only added to default)
```
  <localfile>
    <log_format>syslog</log_format>
    <location>/home/vagrant/*</location>
  </localfile>
```
One wildcard on our home directory to check some log files that we have here.

When we created one folder like `/home/vagrant/myfolder` logcollector detects as log file
```
ossec-logcollector: INFO: (1957): New file that matches the '/home/vagrant/*' pattern: '/home/vagrant/myfolder'.
```
It couldn't be a problem if Wazuh only ignores it but when we put data into that folder and then remove we received alerts on the manager:
```
{"timestamp":"2019-07-29T11:39:00.455+0000","rule":{"level":8,"description":"Log file size reduced.","id":"592","firedtimes":4,"mail":false,"groups":["ossec","attacks"],"pci_dss":["10.5.2","11.4"],"gpg13":["10.1"],"gdpr":["IV_35.7.d"]},"agent":{"id":"001","name":"agent","ip":"10.0.2.15"},"manager":{"name":"localhost.localdomain"},"id":"1564400340.403068","full_log":"ossec: File size reduced (inode remained): '/home/vagrant/myfolder'.","decoder":{"name":"ossec"},"location":"ossec-logcollector"}
```
Also we couldn't monitor logs inside such folder so I think that's better to ignore such folders, thanks for your time.

Regards!.

**Comment by DaveVG1** [comment]
Working on this in [fix-wildcard-logcollector-3769](https://github.com/wazuh/wazuh/compare/fix-wildcard-logcollector-3769)

**Comment by chemamartinez** [comment]
Merged #3788
